cockroachcloud/cmek.md

```diff
@@ -82,6 +82,15 @@ Going forward:
 If the CMEK key is destroyed, the cluster's data can't be recovered or restored from a managed backup in {{ site.data.products.db }} or from a manual backup to the same cluster. It may be possible to restore a manual backup to a new cluster. 
 {{site.data.alerts.end}}
 
+## Rotation of a CMEK key
+
+A CMEK key can be rotated within your KMS platform or within {{ site.data.products.dedicated }}.
+
+- When you rotate a CMEK key within your KMS platform, a new version of the key is created with new key material. In both GCP KMS and AWS KMS, encryption operations automatically uses the active key version, while decryption operations automatically use the key version that was used to encrypt the data. For this reason, {{ site.data.products.dedicated }} does not need any awareness of rotation operations within your KMS platform.
```

Comment:
Same comments/suggestions on these 3 paragraphs as in `cmek-faq.md`. But also, is it worth using an include for this section?

Reply:
Great idea. Incorporated all of the above suggestions except the GCP and AWS URLs (the URIs could change and I don't like to maintain deep links into third-party docs) into the include.